src/openrct2/ride/coaster/meta/BobsleighCoaster.h

```diff
@@ -18,5 +18,13 @@ constexpr const RideTypeDescriptor BobsleighCoasterRTD =
     SET_FIELD(EnabledTrackPieces, (1ULL << TRACK_STRAIGHT) | (1ULL << TRACK_STATION_END) | (1ULL << TRACK_LIFT_HILL) | (1ULL << TRACK_FLAT_ROLL_BANKING) | (1ULL << TRACK_SLOPE) | (1ULL << TRACK_S_BEND) | (1ULL << TRACK_CURVE_SMALL) | (1ULL << TRACK_CURVE) | (1ULL << TRACK_HELIX_SMALL) | (1ULL << TRACK_BRAKES) | (1ULL << TRACK_BLOCK_BRAKES) | (1ULL << TRACK_ON_RIDE_PHOTO)),
     SET_FIELD(ExtraTrackPieces, 0),
     SET_FIELD(TrackPaintFunction, get_track_paint_function_bobsleigh_rc),
+    SET_FIELD(Flags, RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_MAIN | RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_ADDITIONAL |
```

Comment:
Is there any sort of standard set of flags a large amount of rides use. Perhaps we could make this a bit easier to follow if they were grouped.

Reply:
Good idea, you could also group together similar flags, e.g.:

```
RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_MAIN |
RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_ADDITIONAL | RIDE_TYPE_FLAG_HAS_TRACK_COLOUR_SUPPORTS`

to

`RIDE_TYPE_FLAG_HAS_ALL_TRACK_COLOURS `

Reply:
I have created some groups (see third commit).